docs/en/operations/settings/settings.md

```diff
@@ -4144,6 +4144,18 @@ Possible values:
 
 Default value: `0`.
 
+## date_time_overflow_mode {#date_time_overflow_mode}
```

Comment:
Cosmetic: setting name `date_time_overflow_behavior` would be slightly more speaking.